network/router.go

```diff
@@ -101,7 +102,12 @@ func (r *Router) Start() {
 	err := r.host.Listen(func(c Conn) {
 		dst, err := r.receiveServerIdentity(c)
 		if err != nil {
-			log.Error("receive server identity failed:", err)
+			if strings.Contains(err.Error(), "EOF") {
+				// This is probably just a stray connection - ignore it.
+				return
```

Comment:
Do we want to close the connection just to make sure?

Reply:
Something like that? Or will it print an error message if the other end closed the connection and we close it, too?